crates/jstz_cli/src/main.rs

```diff
@@ -59,29 +62,26 @@ enum Command {
         #[arg(name = "data", short, long, default_value = None)]
         json_data: Option<String>,
     },
-    /// Start a REPL session.
+    /// ⚡️ Start a REPL session with jstz's JavaScript runtime.
     Repl {
         /// Sets the address of the REPL environment.
-        #[arg(short, long)]
+        #[arg(value_name = "ADDRESS|ALIAS", short, long)]
         account: Option<AddressOrAlias>,
     },
-    /// Commands related to the logs.
+    /// 🪵 Interact with logs from deployed smart functions.
```

Comment:
```suggestion
    /// 📃 Explore logs from deployed smart functions.
```

Reply:
I prefer 🪵, but like the addition of 'Explore'